fwutil/lib.py

```diff
@@ -633,17 +646,41 @@ def get_status(self):
                             ]
                         )
 
+                        update_status_table.append(
+                            [
+                                chassis_name,
+                                module_name,
+                                chassis_component_name,
```

Comment:
This has to be module_component_name instead chassis_component_name.

Reply:
good catch, fixed in the latest pr.